students/Vladimir12241/hw-3/answers_1.md

```diff
@@ -0,0 +1,27 @@
+- Fx
```

Comment:
Please be aware, that moments when you declare a function and use it are different.

Would you update your answer based on that?

Reply:
uppdated

Reply:
This notation is for ease of verification.
(-FX) - the boundaries of the function itself, does not mean its call